src/api/include/migraphx/migraphx.h

```diff
@@ -518,6 +518,9 @@ MIGRAPHX_C_EXPORT migraphx_status migraphx_onnx_options_set_default_loop_iterati
 MIGRAPHX_C_EXPORT migraphx_status migraphx_onnx_options_set_limit_loop_iterations(
     migraphx_onnx_options_t onnx_options, int64_t value);
 
+MIGRAPHX_C_EXPORT migraphx_status migraphx_onnx_options_set_external_data_path(
```

Comment:
These APIs are auto generated. 
You would need to declare them here : 
https://github.com/ROCm/AMDMIGraphX/blob/eed559feb338b8972335b2aee5d48602d89583b0/src/api/migraphx.py#L324

instead of directly modifying these files.

Reply:
Done.